### PR TITLE
Move Extension Localization Information from Extbase Fluid Book

### PR DESCRIPTION
**Pull request by linawolf**
Resolves https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/issues/720

releases: main, 11.5

### CODE REVIEW

File: Documentation/ExtensionArchitecture/HowTo/Localization/TypoScript.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+.. code-block:: typoscript
+   :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
+
+   [siteLanguage("locale") == "de_CH"]
+       lib.something.value = This site has the locale "de_CH"
+   [end]
+
+   [siteLanguage("title") == "Italy"]
+       lib.languageTitle.value = This site has the title "Italy"
```
Comment (brotkrueml) [inline Documentation/ExtensionArchitecture/HowTo/Localization/TypoScript.rst:38]: Please adjust to the same lib. Also lib.something = TEXT should be added for clarity (outside the condition with a default value).

File: Documentation/ExtensionArchitecture/HowTo/Localization/TypoScript.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+.. code-block:: typoscript
+   :caption: EXT:blog_example/Configuration/TypoScript/setup.typoscript
+
+   plugin.tx_blogexample._LOCAL_LANG.default.comment_header = Remarks
+   plugin.tx_blogexample._LOCAL_LANG.de.comment_header = Bemerkungen
+   plugin.tx_blogexample._LOCAL_LANG.zh.comment_header = 备注
+
+With this, you will overwrite the localization of the term
+:html:`comment_header` for the default language and the languages de and zh
```
Comment (brotkrueml) [inline Documentation/ExtensionArchitecture/HowTo/Localization/TypoScript.rst:68]:
```suggestion
:html:`comment_header` for the default language and the languages "de" and "zh"
```

File: Documentation/ExtensionArchitecture/HowTo/Localization/TypoScript.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+in the blog example.
+
+The :file:`locallang.xlf` files of the extension do not need to be changed for
+this.
+
+:typoscript:`stdWrap.lang`
+==========================
+
+:typoscript:`stdWrap` offers the :ref:`lang <t3tsref:stdwrap-lang>` property
```
Comment (brotkrueml) [inline Documentation/ExtensionArchitecture/HowTo/Localization/TypoScript.rst:77]:
```suggestion
:typoscript:`stdWrap` offers the :ref:`lang <t3tsref:stdwrap-lang>` property,
```

File: Documentation/ExtensionArchitecture/HowTo/Localization/TypoScript.rst [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+with which localized strings can be provided directly by TypoScript.
```
Comment (brotkrueml) [inline Documentation/ExtensionArchitecture/HowTo/Localization/TypoScript.rst:78]:
```suggestion
which can be used to provide localized strings directly from TypoScript.
```

**Review by brotkrueml — CHANGES_REQUESTED** [review]

**Comment by linawolf** [comment]
This PR has been split up in smaler, more manageable PRs

**Pull request closed without merging**